x-pack/plugins/alerting/server/routes/find_rules.ts

```diff
@@ -20,7 +21,7 @@ const querySchema = schema.object({
   default_search_operator: schema.oneOf([schema.literal('OR'), schema.literal('AND')], {
     defaultValue: 'OR',
   }),
-  search_fields: schema.maybe(schema.arrayOf(schema.string())),
+  search_fields: schema.maybe(schema.oneOf([schema.arrayOf(schema.string()), schema.string()])),
```

Comment:
I had to add this back to support `?search_fields=foo`, which was caught in our tests.